opennms-web-api/src/main/java/org/opennms/web/svclayer/model/RelativeTimePeriod.java

```diff
@@ -48,6 +48,16 @@
     
     static {
         s_defaultPeriods = new RelativeTimePeriod[] {
+                new RelativeTimePeriod("1_hour", "Last Hour", Calendar.HOUR,
```

Comment:
can we make them only use one line? This looks weird. Can we also name them `last_1_hour` or something like that?